### DISCUSSION TITLE
Issue templates

### DISCUSSION THREAD

**Discussion by aleksasiriski** [mysticaltech/terraform-hcloud-kube-hetzner#575] [Ideas]
We should make a simple template that uses three ` and terraform syntax for kube.tf files so that people always send their configs instead of us asking for them always.. I don't have enough privileges on the repo to do it myself so @mysticaltech will have to do it.

**Comment by mysticaltech** [comment]
@aleksasiriski Great idea! I've been thinking of that too, glad you reminded me. I added the templates, and also escalated your privileges because I trust your good judgment, so from now on you can do these things. But if it's an important change, please let me know first.

**Reply by aleksasiriski** [reply]
Of course I will consult you first, it's your project after all

**Reply by mysticaltech** [reply]
It's our project now, I just started it. Thank you valuable input always.

**Reply by aleksasiriski** [reply]
Btw I don't have access to Features > Issue templates

I just wanted to add ```terraform to kube.tf section so that it's always color coded (much easier to see if something is commented or not)

**Reply by mysticaltech** [reply]
You have the "maintain" privileges now, too bad it can do this. I will take care of it, I understood what you want. It's a good idea.

**Reply by mysticaltech** [reply]
@aleksasiriski It's done, and great initiative, this should simplify our lives a lot!

**Reply by mysticaltech** [reply]
For reference, it turned out to be just files that needed to be created over at https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/tree/master/.github/ISSUE_TEMPLATE.

**Reply by aleksasiriski** [reply]
Oh cool that makes sense!
